datumaro/plugins/sampler/random_sampler.py

```diff
@@ -18,13 +18,19 @@ class RandomSampler(Transform, CliPlugin):
     Sampler that keeps no more than required number of items in the dataset.|n
     |n
     Notes:|n
-    - Items are selected uniformly|n
-    - Requesting a sample larger than the number of all images will
-    return all images|n
+    |s|s- Items are selected uniformly|n
+    |s|s- Requesting a sample larger than the number of all images will
+        return all images|n
     |n
     Example: select subset of 20 images randomly|n
+
+    .. code-block::
+
     |s|s%(prog)s -k 20 |n
     Example: select subset of 20 images, modify only 'train' subset|n
+
+    .. code-block::
+
```

Comment:
These docstrings are displayed in the command line help. I think, we can either remove them from the API docs, because it's already written in CLI docs, or find another method of displaying this info in API docs.

Reply:
Thanks for the tip! I added a `.replace` to `datumaro/cli/util/__init__.py` so that I could leave `.. code-block::` in the source code, but it didn't show up in CLI help. Is such a decision acceptable?

Reply:
I think so.